common/util/http_util.go

```diff
@@ -30,6 +30,7 @@ import (
 
 	errorType "github.com/dragonflyoss/Dragonfly/common/errors"
 
+	"crypto/tls"
```

Comment:
That's the reason why you cannot pass the CI. In fact, we recommend the imported packages should be cut into three parts and separate them with a blank line
+ standard packages
+ the packages from Dragonfly project
+ the packages from the third party projects